src/antlr/Parser.g

```diff
@@ -795,9 +795,30 @@ columnConstraints returns [ColumnConstraints.Raw constraints]
     ;
 
 columnConstraint returns [ColumnConstraint columnConstraint]
-    : funcName=ident '(' k=ident ')' op=relationType t=value { $columnConstraint = new FunctionColumnConstraint.Raw(funcName, k, op, t.getText()).prepare(); }
-    | funcName=ident '(' k=ident ')' { $columnConstraint = new UnaryFunctionColumnConstraint.Raw(funcName, k).prepare(); }
-    | k=ident op=relationType t=value { $columnConstraint = new ScalarColumnConstraint.Raw(k, op, t.getText()).prepare(); }
+    @init { List<String> arguments = new ArrayList<>(); }
+    : K_NOT K_NULL
```

Comment:
Not for this PR, and don't know if worth doing, but here it goes yet another crazy idea... What about something like:
```
(K_NOT)? funcName=ident
```

With the `K_NOT` optional we can "negate" whatever the func does. This would allow us to reuse the `NOT` notation for all other constraints. A crazy example would be:
```
ALTER TABLE ks.tb ADD val3 text CHECK NOT JSON
```